third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -214,9 +214,12 @@ BraveSessionCache::BraveSessionCache(ExecutionContext& context)
   CHECK(h.Init(reinterpret_cast<const unsigned char*>(&session_key_),
                sizeof session_key_));
   CHECK(h.Sign(domain, domain_key_, sizeof domain_key_));
-  settings_client_ = GetContentSettingsClientFor(&context, true);
-  if (settings_client_ != nullptr) {
-    auto raw_farbling_level = settings_client_->GetBraveFarblingLevel(
+  const uint64_t* fudge = reinterpret_cast<const uint64_t*>(domain_key_);
```

Comment:
Changes in this file not meant to be here, need to revert it.
CC: @bsclifton

Reply:
Reverted it as clean up commit(Since same file mentioned in two commit, so ignored `git revert`, now you can start the CI.

@bsclifton